LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Fastify
+Copyright (c) 2024 Fastify
```

Comment:
If you want to make it consistent with fastify repo, it should be an interval, see https://github.com/fastify/fastify/blob/main/LICENSE